terrautils/extractors.py

```diff
@@ -425,19 +558,10 @@ def find_timestamp(self, text=None):
                 timestamp = self.extract_timestamp(self.dataset_metadata['name'])
 
         if timestamp is None and not self.experiment_metadata is None:
-            if 'observationTimeStamp' in self.experiment_metadata:
-                timestamp = self.experiment_metadata['observationTimeStamp']
+            timestamp = __internal__.case_insensitive_find(self.experiment_metadata, 'observationTimeStamp')
 
         if timestamp is None:
-            _zero = datetime.timedelta(0)
-            class TZ(datetime.tzinfo):
-                """Internal class used to format UTC timestamps properly
-                """
-                def utcoffset(self, _dt):
-                    return _zero
-                def dst(self, _dt):
-                    return _zero
-            timestamp = datetime.datetime.utcnow(tz=TZ()).isoformat()
+            timestamp = datetime.datetime.utcnow().isoformat()
```

Comment:
is the code for TZ no longer needed?

Reply:
That is correct. The use of it was a copy&paste error.